### ISSUE TITLE
Add to root MQTT topic panel SN

### ISSUE THREAD

**Issue by TheKito** [ParadoxAlarmInterface/pai#434]
Hi, 

The idea is to add at MQTT root topic, panel serial number, to handle multiple panels at the same time without create a full custom config for each 

I do it in a dirty way editing file paradox/connections/ip/commands.py

and add at file header:
`from paradox.config import config as cfg, get_limits_for_type`

then add at line 130 (before logger.info):         

`cfg.MQTT_BASE_TOPIC = "PDX" + binascii.hexlify(response.ip_module_serial).decode("utf-8").upper()`

I'm sure it wasn't the best way to do it, but I only know a little of Python.

Thanks 
Regards

**Comment by TheKito** [comment]
yes, but i was thinking in the context of have multiple alarm panels in cloud hosted broker, to make a swan open alternative

**Comment by yozik04** [comment]
but everybody wants to have different MQTT_BASE_TOPIC. You want serial number, somebody wants street name and so on.
